miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -909,6 +909,9 @@ protected PoolDto createDtoFromParent(LibraryDilution item) {
       dto.setAlias(item.getLibrary().getAlias() + "_POOL");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "_POOL" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)